php/blocks/migration/class-post-type.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Post_Type.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2020, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Migration;
+
+use WP_Post;
+use WP_Query;
+
+/**
+ * Class Post_Type
+ */
+final class Post_Type {
```

Comment:
We haven't used `final` anywhere else. in Block Lab. Happy to include it – I can't imagine this class ever needing a child, but just wanted to call that out for consistency.

Reply:
Thanks, good point. Yeah, it's probably not needed. We were using it with the AMP plugin so classes couldn't be extended, but I don't think people are going to be extending this.